sbi/utils/sbiutils.py

```diff
@@ -27,9 +28,9 @@ def forward(self, tensor):
 def match_shapes_of_theta_and_x(
     theta: Union[Sequence[float], float],
     x: Union[Sequence[float], float],
-    x_o: torch.Tensor,
+    x_o: Tensor,
     correct_for_leakage: bool,
-) -> (torch.Tensor, torch.Tensor):
+) -> (Tensor, Tensor):
```

Comment:
Do you not have complaints from `pyright` that this must be `Tuple[Tensor, Tensor]`?